.github/workflows/build.yml

```
@@ -0,0 +1,94 @@
+name: Build
+
+on:
+    push:
+        branches: [main, next, master, 2.5, 3.4, 4.3, 5.1, 6.2]
+    pull_request:
+    release:
+        types: [created]
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        name: Build and test
+        strategy:
+            matrix:
+                php: [7.3, 7.4]
+                composer-flags: [""]
+                composer-extras: [""]
+                stability: ["stable"]
+                include:
+                    -   php: 7.3
+                        publish-phar: true
+                    -   php: 7.3
+                        composer-flags: --prefer-lowest
+                    -   php: 7.3
+                        composer-extras: "symfony/event-dispatcher ^3.4 symfony/contracts ^2.0"
```

Shouldn't it be written as such?

```suggestion
                        composer-extras: "symfony/event-dispatcher^3.4 symfony/contracts^2.0"
```

---

I kept what was there in the travis config.